src/common/sessiond-comm/relayd.h

```diff
@@ -191,6 +191,15 @@ struct lttcomm_relayd_create_session_2_4 {
 	uint32_t snapshot;
 } LTTNG_PACKED;
 
+struct lttcomm_relayd_create_session_2_11 {
+	uint32_t session_name_len;
+	uint32_t hostname_len;
+	uint32_t live_timer;
+	uint32_t snapshot;
```

Comment:
Can be change to `uint8_t`.

Reply:
On the sender side the snapshot value is contained in a unsigned int. Not sure why. Still I used the !! trick to ensure that we bound our self to a byte.